Ansible/ansible_collections/jfrog/platform/roles/artifactory/tasks/upgrade.yml

```diff
@@ -27,6 +27,7 @@
     owner: "{{ artifactory_user }}"
     group: "{{ artifactory_group }}"
     creates: "{{ artifactory_untar_home }}"
+    notify: stop artifactory
```

Comment:
@ZwareBear can you please change this notify inline with when ( indentation issue)

```
  when: (download_artifactory is succeeded) and (not ansible_check_mode)
  notify: Stop artifactory
```

Reply:
Good catch! Thanks.